style.css

```diff
@@ -1,7 +1,315 @@
-/* reset*/
+/* inter-regular - latin */
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  src: url('https://fonts.googleapis.com/css?family=Inter');
+}
+
+/* reset */
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* body settings */
+body {
+  background: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+main {
+  border: red;
+}
+
+.menu-button {
+  border: 3px solid black;
+  background: white;
+  width: 24px;
+}
+
+.headline {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  font-weight: 700;
+  font-size: 40px;
+}
```

Comment:
- [ ] You did great work on implementing the requested changes by the previous reviewer 👏 . However, I think it would be cool if you reduce a bit the `font size` on your headline to something like `27 px`.

Reply:
@ridwanediallo, great and thanks. Change done.